src/main/java/mate/academy/service/impl/AuthenticationServiceImpl.java

```diff
@@ -16,11 +16,16 @@
 
     @Override
     public User login(String email, String password) throws AuthenticationException {
+        if (userService.findByEmail(email).isEmpty()) {
```

Comment:
Ideally, make single if clause to shorten the code

Reply:
Ok, i will do it